app/models/vote.rb

```diff
@@ -0,0 +1,22 @@
+class Vote < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :review
+
+  validates :user, presence: true
+  validates :review, presence: true
+  validates :user_id, uniqueness: {
+    scope: :review_id, message: "User can only vote once per review."
+    }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.